test/index.spec.js

```diff
@@ -228,21 +228,21 @@ describe('FlipMove', () => {
     });
   });
 
-  describe('duration propType', () => {
-    it('applies a string that can be converted to an int', () => {
-      renderedComponent.setState({ duration: '10' });
-      expect(consoleStub).to.not.have.been.called;
-    });
-
+  describe('duration prop runtime checking', () => {
     it('applies a bogus string', () => {
       renderedComponent.setState({ duration: 'hi' });
       expect(consoleStub).to.have.been.calledOnce;
     });
 
     it('applies an array prop and throws', () => {
-      renderedComponent.setState({ duration: [10] });
+      renderedComponent.setState({ duration: ['hi'] });
       expect(consoleStub).to.have.been.called;
     });
+
+    it('applies a string that can be converted to an int', () => {
```

Comment:
I needed to move the "correct" testcase down, because all these tests persist state between cases, and some of them rely on that. This is not good in fact, but it's a separate issue.